### ISSUE TITLE
How do you set a null value in orientdb-server-config.xml

### ISSUE THREAD

**Issue by wouterv** [orientechnologies/orientdb#5364]
I want to disable the database backup before syncing. 

According to this isses https://github.com/orientechnologies/orientdb/issues/3773 it can be done:

"Starting from 2.1.3 you can define your own backup directory by setting distributed.backupDirectory to the right path. Null to avoid backup at all."

How do you set a null value?

I tried:
```
        <entry value="null" name="distributed.backupDirectory"/>
```
this just creates a directory with the name "null"

I also tried:
```
        <entry value="" name="distributed.backupDirectory"/>
```
But that gives an error as it wants to override it's own directory.

**Comment by lvca** [comment]
Do you have any logs of this problem?

**Comment by wouterv** [comment]
This is the error in case you leave the value empty:

```
2015-11-23 15:25:11:570 SEVER [prod2] error on moving existent database 'beeyond' located in '/opt/orientdb/databases/beeyond' to '/opt/orientdb/databases/beeyond'. Try to move the database directory manually and retry [OHazelcastPlugin][prod2] Error on starting distributed plugin
com.orientechnologies.orient.server.distributed.ODistributedException: Error on moving existent database 'beeyond' located in '/opt/orientdb/databases/beeyond' to '/opt/orientdb/databases/beeyond'. Try to move the database directory manually and retry
        at com.orientechnologies.orient.server.hazelcast.OHazelcastPlugin.backupCurrentDatabase(OHazelcastPlugin.java:1007)
        at com.orientechnologies.orient.server.hazelcast.OHazelcastPlugin.requestDatabase(OHazelcastPlugin.java:954)
        at com.orientechnologies.orient.server.hazelcast.OHazelcastPlugin.installDatabase(OHazelcastPlugin.java:893)
        at com.orientechnologies.orient.server.hazelcast.OHazelcastPlugin.installNewDatabases(OHazelcastPlugin.java:1448)
        at com.orientechnologies.orient.server.hazelcast.OHazelcastPlugin.startup(OHazelcastPlugin.java:185)
        at com.orientechnologies.orient.server.OServer.registerPlugins(OServer.java:979)
        at com.orientechnologies.orient.server.OServer.activate(OServer.java:346)
        at com.orientechnologies.orient.server.OServerMain.main(OServerMain.java:41)
```

**Comment by lvca** [comment]
Ok, this was already fixed in 2.1.6. We're almost ready to release 2.1.6 in the next hours.

**Comment by wouterv** [comment]
So, what is going to be the correct syntax?

**Comment by lvca** [comment]
With 2.1.6 what you did is correct. With your release I suggest to point it to a folder where OrientDB can write for sure. For example `/tmp` could be a good setting.

**Comment by wouterv** [comment]
I guess I wasn't clear enough. I do not want a backup at all. It is too big.

**Comment by lvca** [comment]
There is no way by configuration to pass a null. However in 2.1.6 has been fixed. We've released 2.1.6 right now. Could you please upgrade and keep this setting as empty?

**Comment by wouterv** [comment]
Confirmed fixed.
